packages/ui5-application-inquirer/src/prompts/prompts1.ts

```diff
@@ -131,7 +131,24 @@ export function getTargetFolderPrompt(targetDir: string, validateFioriAppFolder?
             breadcrumb: t('prompts.targetFolder.breadcrumb')
         },
         default: (answers: UI5ApplicationAnswers) => answers.targetFolder || targetDir,
-        validate: async (target, { name = '' }: UI5ApplicationAnswers): Promise<boolean | string> => {
+        additionalMessages: (target: string, answers: UI5ApplicationAnswers) => {
+            const { name = '' } = answers;
+            const { namespace = '' } = answers;
+            let message;
+            // Windows path length validation
+            if (process.platform === 'win32') {
+                const combinedLength = `${target}\\${namespace || ''}\\${name}`.length;
+                if (combinedLength >= 256) {
+                    message = {
+                        message: t('validators.windowsFolderPathTooLong', { length: combinedLength }),
```

Comment:
This is a validation text but its being used in an additional message output. They are different. Ones blocking and one insnt.

Reply:
I believe there is a Windows setting that allows long paths. So I wiwn't want to block the user.